### DISCUSSION TITLE
SQLiteDialect_pysqlite cannot be used as a data source for security reasons.

### DISCUSSION THREAD

**Discussion by dev-x-master** [apache/superset#20731] [Q&A / Help]
Howdy y'all! I've a error in the middle of using superset database. I gotta add new database from my local storage, but I've got a error as follow in screenshot. How can I solve this? Thank you in advance.
![superset-sql](https://user-images.githubusercontent.com/105999803/179361746-dffd9173-95ae-4a2c-806f-25f66b89b628.png)

**Comment by igitur** [comment]
I just followed the [instructions](https://hub.docker.com/r/apache/superset) for starting up the latest docker image. The instructions include a step to load the examples database (`docker exec -it superset superset load_examples`), but even this out-of-the-box database fails to load with this same error.
